app/src/main/java/com/mashup/mobalmobal/ui/profile/presenter/ProfileViewModel.kt

```diff
@@ -3,52 +3,143 @@ package com.mashup.mobalmobal.ui.profile.presenter
+            .subscribeOnIO()
+            .subscribeWithErrorLogger { userDto ->
+                userDto.data?.let { _userDtoSubject.onNext(it) }
+            }.addToDisposables()
     }
 
-    fun requestProfile(userId: String) {
-        profileRepository.getProfile(userId)
-            .toFlowable()
+    private val _donationsSubject: BehaviorSubject<List<ProfileItem>> =
+        BehaviorSubject.createDefault(emptyList())
+
+    init {
+        requestDonations()
             .subscribeOnIO()
-            .doOnSubscribe { _loadingSubject.onNext(true) }
-            .subscribeWithErrorLogger { response ->
-                with(response) {
-                    if (result != Constants.NETWORK_SUCCESS) {
-                        throw IOException(data.message)
-                    }
-                    _profileSubject.onNext(data.toProfileItems())
-                }
-                _loadingSubject.onNext(false)
+            .subscribeWithErrorLogger {
+                _donationsSubject.onNext(it)
+            }
+            .addToDisposables()
+    }
+
+    private val _itemSubject: BehaviorSubject<List<ProfileItem>> = BehaviorSubject.create()
+    val itemSubject: Observable<List<ProfileItem>> = _itemSubject.distinctUntilChanged()
+
+    init {
+        _userDtoSubject.combineLatest(_donationsSubject)
+            .distinctUntilChanged()
+            .map { listOf(it.first.toProfileItem()) + it.second }
+            .subscribeWithErrorLogger {
+                _itemSubject.onNext(it)
             }
             .addToDisposables()
     }
 
+    private fun requestDonations() =
+        requestMyCreateDonations()
+            .zipWith(requestMyDonated())
+            .zipWith(requestMyClosed())
+            .map {
+                val createDonations =
+                    it.first.first.toProfileItems(R.string.profile_header_donation_request)
+                val donatedDoantions =
+                    it.first.second.toProfileItems(R.string.profile_header_donated)
+                val closedDonations =
+                    it.second.toProfileItems(R.string.profile_header_donation_closed)
+
+                mutableListOf<ProfileItem>().apply {
+                    addAll(
+                        Triple(
+                            createDonations.getDonationCount(),
+                            donatedDoantions.getDonationCount(),
+                            closedDonations.getDonationCount()
+                        ).toSummaryProfileItem()
+                    )
+
+                    addAll(createDonations + donatedDoantions + closedDonations)
+                }
+            }
+
+    private fun requestMyCreateDonations() =
+        profileRepository.getMyPosts(STATUS_DONATION_BEFORE)
+            .zipWith(profileRepository.getMyPosts(STATUS_DONATION_INPROGRESS))
+            .map { pair ->
+                val beforeDonation = pair.first.data.posts
+                val inProgressDonation = pair.second.data.posts
```

Comment:
```suggestion
            .map { (myBeforePosts, myInProgressPosts) ->
                val beforeDonation = myBeforePosts.data.posts
                val inProgressDonation = myInProgressPosts.data.posts
```
pair보단 의미있는 변수명이 가독성에 더 좋을 것 같아요

Reply:
좋습니다~!